cuizinart/metadata_schema.py

```diff
@@ -1,10 +1,12 @@
 import os
-from flask_migrate import Migrate
+#from flask_migrate import Migrate
 from flask_sqlalchemy import SQLAlchemy
 from flask import Flask
 from marshmallow import fields
 from flask_marshmallow import Marshmallow
 from settings import *
+from flask_script import Manager
```

Comment:
I assume you are importing this to get `flask-script db init` to work, because `flask db init` doesn't work on your machine. However, flask-script is deprecated in favor of flask's built-in CLI, so we should try getting it to work on your machine, too. On both tuna and on my machine, the CLI (`flask db init`) works...